frontend/test/metabase/scenarios/models/reproductions/20624-model-metadata-should-override-column-settings.cy.spec.js

```diff
@@ -0,0 +1,42 @@
+import { restore } from "__support__/e2e/cypress";
+import { openDetailsSidebar } from "../helpers/e2e-models-helpers";
+
+const renamedColumn = "TITLE renamed";
+
+const questionDetails = {
+  name: "20624",
+  dataset: true,
+  native: { query: "select * from PRODUCTS limit 2" },
+  visualization_settings: {
+    column_settings: { '["name","TITLE"]': { column_title: renamedColumn } },
+  },
+};
+
+describe.skip("issue 20624", () => {
+  beforeEach(() => {
+    cy.intercept("PUT", "/api/card/*").as("updateCard");
+
+    restore();
+    cy.signInAsAdmin();
+
+    cy.createNativeQuestion(questionDetails, { visitQuestion: true });
+  });
+
+  it("models metadata should override previously defined column settings (metabase#20624)", () => {
+    openDetailsSidebar();
+    cy.findByText("Customize metadata").click();
+
+    // Open settings for this column
+    cy.findByText(renamedColumn).click();
```

Comment:
@flamber I made it fail on the first step - because the column name in the model's metadata doesn't respect the renamed column.